yt_dlp/_globals.py

```diff
@@ -0,0 +1,28 @@
+from collections import defaultdict
+
+# Internal only - no backwards compatibility guaranteed
```

Comment:
Can be made public imo. `get_plugin_dirs` etc are very unpythonic imo and can be removed in favor of `plugins.plugin_dirs.get()` etc

Reply:
Agreed. Though given the speed of iteration in the plugin space (downloader plugins? more options?) it might be worth it to explicitly state that no backwards compatibility is guaranteed, though we can still try our best.